### PR TITLE
Add Linux client install steps

### PR DESCRIPTION
**Pull request by PCNZ**
Some initial instructions for users covering how to install on Linux clients

### CODE REVIEW

File: content/docs/deployment/clients/_index.md [hunk 1]
```diff
@@ -188,6 +188,47 @@ The service can be uninstalled with the following command:
 
 Confirm with `ps -eaf | grep velo` and `sudo launchctl list | grep velo`.
 
+### Linux
+
+The dpkg (Debian)or rpm (Red Hat/CentOS/Fedora) tools can be used to install Velociraptor on
+Linux clients after creating an appropriate package. The following command installs binary & config to /usr/local/sbin.
+Persistence is via launchd (check with `ps -eaf | grep velo` and
```
Comment (scudette) [inline content/docs/deployment/clients/_index.md]: That's for macos no?

File: content/docs/deployment/clients/_index.md [hunk 2]
```diff
@@ -188,6 +188,47 @@ The service can be uninstalled with the following command:
...
+```shell
+# velociraptor --config client.config.yaml debian client
+```
+Install
+```shell
+# sudo dpkg -i velociraptor.deb
+```
+Remove
+```shell
+# sudo apt-get remove velociraptor
```
Comment (scudette) [inline content/docs/deployment/clients/_index.md]: The client package is called velociraptor_client

**Review by scudette — CHANGES_REQUESTED** [review]
Thanks!

File: content/docs/deployment/clients/_index.md (hunk 1 above)
Comment (PCNZ) [reply content/docs/deployment/clients/_index.md]: Removed, copied by mistake from the Mac section

File: content/docs/deployment/clients/_index.md (hunk 2 above)
Comment (PCNZ) [reply content/docs/deployment/clients/_index.md]: Updated should be clearer now.

**Review by scudette — APPROVED** [review]
Thanks 😊

**Pull request merged**